src/views/onboarding/components/OnBoardingProductStepDelegates.tsx

```diff
@@ -41,6 +41,12 @@ export function OnBoardingProductStepDelegates({ product, legal, forward, back }
   >('delegateFormIds', []);
   const { t } = useTranslation();
 
+  useEffect(() => {
+    if (isAuthUser) {
+      setPeopleErrors({});
```

Comment:
nope, you have to remove just the error of the first legal, not all the errors :)

to test it, insert 2 certified legals, having incorrect certified data, then click on the checkBox, just the error on the first legal should disappear ;)